1_IGT_PP/Code/R/1_Analyses/fitting_ORL_lineartime.Rmd

```diff
@@ -26,7 +26,7 @@ library(haven)
 
 ## Load Data
 ```{r}
-stan_data <- readRDS(here("1_IGT_PP", "Data", "1_Preprocessed", "stan_ready_ORL_joint_retest.rds"))
+stan_data <- readRDS(here("1_IGT_PP", "Data", "1_Preprocessed", "stan_ready_ORL_IGT.rds"))
```

Comment:
I think this name here should be `stan_ready_IGT_ORL.rds` instead of this? e.g.,: 

```suggestion
stan_data <- readRDS(here("1_IGT_PP", "Data", "1_Preprocessed", "stan_ready_IGT_ORL.rds"))
```